CHANGES_NEXT_RELEASE

```diff
@@ -11,3 +11,4 @@
 - [cygnus][doc] Add Orion-Cygnus-Kafka integration example (#1132)
 - [cygnus-ngsi][hardening] Decrease to DEBUG the logging level of batching related traces (#1126)
 - [cygnus-common][bug] Release connection upon persistence error (#1139)
+- [cygnus-common][bug] Insert NULL in CKAN columns when an empty ("", null, {} or []) attribute value is notified (#1144)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1148/commits/c8e06e24f337c9ee971b6021cfd955f3fd1e7aea